### PR TITLE
fix broken visaservice-go test

### PR DESCRIPTION
**Pull request by mkolehmainen**

### CODE REVIEW

File: visaservice/core/pkg/vservice/vsinst-thrift.go [hunk 1]
```diff
@@ -244,13 +244,13 @@ func vsapiTrafficDescToIpTraffic(pd *vsapi.TrafficDesc) *snip.Traffic {
 //
 
 // returns API key
-func (vs *VSInst) BackDoorInstallAPIKeyForUnitTest(node_addr netip.Addr, node_name string) (string, error) {
-	return vs.BackDoorInstallAPIKeyForUnitTestExp(node_addr, node_name, time.Now().Add(5*time.Minute))
+func (vs *VSInst) BackDoorInstallAPIKeyForUnitTest(node_addr netip.Addr, node_name, vssAddr string) (string, error) {
```
Comment (owenrhysthomas) [inline visaservice/core/pkg/vservice/vsinst-thrift.go:230]: Do you still want `node_name string`? I'm not very familiar with Go syntax, I just noticed that node_name is now the only parameter to a function that appears alone in the function header.

**Review by owenrhysthomas — APPROVED** [review]
Look good, assuming removing `string` after that parameter was correct.

File: visaservice/core/pkg/vservice/vsinst-thrift.go (hunk 1 above)
Comment (mkolehmainen) [reply visaservice/core/pkg/vservice/vsinst-thrift.go:230]: ah in go you can put the type at the end and use commas if the type is the same.  So foo(a, b string) means same as foo(a string, b string).

Comment (owenrhysthomas) [reply visaservice/core/pkg/vservice/vsinst-thrift.go:230]: Got it, that makes sense. I figured it was an optional type thing, I didn't realize that you could use commas if they were the same! That's cool.

**Pull request merged**